README.md

```diff
@@ -129,3 +130,7 @@ One key purpose of PAI is to support the highly diversified requirements from ac
 PAI operates in an open model. It is initially designed and developed by [Microsoft Research (MSR)](https://www.microsoft.com/en-us/research/group/systems-research-group-asia/) and [Microsoft Search Technology Center (STC)](https://www.microsoft.com/en-us/ard/company/introduction.aspx) platform team.
 We are glad to have [Peking University](http://eecs.pku.edu.cn/EN/), [Xi'an Jiaotong University](http://www.aiar.xjtu.edu.cn/), [Zhejiang University](http://www.cesc.zju.edu.cn/index_e.htm), and [University of Science and Technology of China](http://eeis.ustc.edu.cn/) join us to develop the platform jointly.
 Contributions from academia and industry are all highly welcome.
+
+#### How to upgrade
+PAI officially supports upgrade, since the release v0.10.1.
+Follow the document [Upgrade to v0.10](./docs/upgrade/upgrade_to_v0.10.md) for more details.
```

Comment:
Maybe give user a directory view to `./docs/upgrade`, so when we have upgrade doc to v0.11, we do not have to remember update this README.md

Reply:
Well, we will need to update this README.md.
As the title says, it's `Upgrade to v0.11`.

Reply:
Ah, I see what you meant.